tools/isobuild/linker.js

```diff
@@ -612,6 +612,17 @@ var getHeader = function (options) {
     chunks.push("var " + options.packageVariables.join(', ') +
                 ";\n\n");
   }
+  if (options.combinedServePath || options.name) {
+    chunks.push("/* Package reflection */\n");
+    if (options.name) {
+      chunks.push("var PackageName = " + JSON.stringify(options.name) + ";\n");
+    }
+    if (options.combinedServePath) {
+      chunks.push("var PackagePath = " +
```

Comment:
But frankly the main use case for me was for dynamic loading (#6203) of modules. So that I could lookup into the manifest file and see dependencies (one other change I added to the manifest file for each entry there). But then I discovered that it is simply easier to list directly all dependencies in the header of the file. Instead of having a key to resolve in manifest to load dependencies.